docs/content/reference/api-usage.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NGINX Service Mesh API
+description: "Instructions for using the NGINX Service Mesh API."
```

Comment:
```suggestion
description: "Instructions for interacting with the NGINX Service Mesh API."
```

Small change, but "using" is slightly more ambiguous: it could be referring to consuming information from the API or making changes with it.